<launch>
  <arg name="mav_name" default="pelican"/>
  <arg name="world_name" default="basic"/>
  <arg name="enable_logging" default="false" />
  <arg name="enable_ground_truth" default="true" />
  <arg name="log_file" default="$(arg mav_name)" />
  <arg name="debug" default="false"/>
  <arg name="gui" default="true"/>
  <arg name="paused" default="true"/>
  <!-- The following line causes gzmsg and gzerr messages to be printed to the console
      (even when Gazebo is started through roslaunch) -->
  <arg name="verbose" default="false"/>
    <!-- <node name="position_controller_node" pkg="rotors_control" type="lee_position_controller_node" output="screen">
      <rosparam command="load" file="$(find rotors_gazebo)/resource/lee_controller_$(arg mav_name).yaml" />
      <rosparam command="load" file="$(find rotors_gazebo)/resource/$(arg mav_name).yaml" />
      <remap from="odometry" to="odometry_sensor1/odometry"/>
    </node>   -->
      
    <node name="position_controller_node" pkg="rrc_control" type="pid_position_controller_node" output="screen">
      <rosparam command="load" file="$(find rrc_control)/resources/pid_controller_$(arg mav_name).yaml" />
      <rosparam command="load" file="$(find rrc_control)/resources/$(arg mav_name).yaml" />
    </node>  
<!--     <node name="position_controller_node" pkg="rrc_control" type="asmc_position_controller_node" output="screen">
      <rosparam command="load" file="$(find rrc_control)/resources/asmc_$(arg mav_name).yaml" />
      <rosparam command="load" file="$(find rrc_control)/resources/$(arg mav_name).yaml" />
      <remap from="odometry" to="odometry_sensor1/odometry" />
      <remap from="/odometry/pose" to="mocap_opti/pose" />
    </node>  
 -->
<!--     <node name="position_controller_node" pkg="rrc_control" type="rsb_position_controller_node" output="screen">
      <rosparam command="load" file="$(find rrc_control)/resources/rsb_$(arg mav_name).yaml" />
      <rosparam command="load" file="$(find rrc_control)/resources/$(arg mav_name).yaml" />
      <remap from="odometry" to="odometry_sensor1/odometry" />
    </node>  
 -->

</launch>
